proto/session/service_streaming.proto

```diff
@@ -27,13 +27,15 @@ message RequestWrapper {
   oneof request {
     GetPublicKeyRequest get_public_key_request = 1;
     InvokeRequest invoke_request = 2;
+    GetEndorsedEvidenceRequest get_endorsed_evidence_request = 3;
```

Comment:
Is the plan still to eventually remove the service definition (below) from this file?

Reply:
Yes, we will have individual service definitions per untrusted application. And the prod teams will be able to define their own services.

The only reason to keep the service definition below could be to be used in our examples. But in this case each example can have it's own service definition.